server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader9to10.java

```diff
@@ -111,6 +111,31 @@
  * {@link #dropSortedMapWALFiles(ServerContext)}. For more information see the following issues:
  * <a href="https://github.com/apache/accumulo/issues/2117">#2117</a> and
  * <a href="https://github.com/apache/accumulo/issues/2179">#2179</a>
+ *
+ * The following methods were created for External compactions:
+ * {@link #createExternalCompactionNodes(ServerContext)}, {@link #setMetaTableProps(ServerContext)}
+ *
+ * Improvements to the metadata and root tables were made in this version. See pull request
+ * <a href="https://github.com/apache/accumulo/pull/1174">#1174</a> for more details. The
+ * {@link #upgradeRootTabletMetadata(ServerContext)} method was added for this change. This change
+ * allowed other changes to how volumes were stored in the metadata and have Accumulo always call
+ * the volume chooser for new tablet files. This change was facilitated with the
+ * {@link #upgradeDirColumns(ServerContext, Ample.DataLevel)} method and done in
+ * <a href="https://github.com/apache/accumulo/pull/1389">#1389</a>
```

Comment:
Again, we don't need to document the changes here. If anything here about these methods is useful, it would be more useful on the actual methods, instead of at the class level.

Reply:
I can move javadoc to the methods. But I was trying to link the methods at a higher level and some of them have multiple methods and called over the different upgrade steps.